specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -2723,6 +2723,51 @@
         }
       }
     },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingProfiles/{billingProfileName}/invoices/{invoiceName}/transactions": {
+      "get": {
```

Comment:
>"get" [](start = 6, length = 5)

Should contain the x-ms-pageable flag to let clients know how to follow nextLinks.  [https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-pageable](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-pageable)

Reply:
Sure! done.